content/workflows/text-semantic-viewer.md

```diff
@@ -0,0 +1,12 @@
++++
+title= "Semantic search"
+images =  ["/workflow_images/text-semantic-search.png", "/workflow_images/text-semantic-search-widget.png"]
+type = "workflows"
+blog_link =  ""
+video = ""
+download = "text-semantic-search.ows"
+workflows = ["Text Mining"]
+weight = 655
++++
+
+We can find relevant parts of a document by searching for exact words or parts of documents with similar meanings. The Semantic Viewer widget in this workflow search for pertinent sentences of the documents by comparing the meaning of words from the Word List widget to the meaning of sentences in the text via SBERT text embeddings. The widget lists documents with scores that reveal the documents' relevance, shows selected documents, and highlights relevant parts.
```

Comment:
> The Semantic Viewer widget in this workflow search for

searches

> The widget lists documents with scores that reveal the documents' relevance, shows selected documents, and highlights relevant parts.

The widget ranks documents by scores measuring their relevance, shows selected documents, and highlights the relevant parts.